src/components/icons/LogoFesbalIcon.tsx

```diff
@@ -1,6 +1,6 @@
 const LogoFesbalIcon = () => {
     return (
-        <svg width="225" height="230" viewBox="0 0 225 230" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <svg width="188" height="190" viewBox="0 0 225 230" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Check the icon it's showing correctly because you didn't update the viewBox proportion and it's possible the icon doesn't respect the aspect ratio